package.json

```diff
@@ -12,6 +12,7 @@
   ],
   "packageManager": "pnpm@9.5.0",
   "scripts": {
+    "preinstall": "npx only-allow pnpm",
```

Comment:
Why do we need this @geekyharsh05?

Reply:
To allow only pnpm as the package manager. If accidentally someone runs yarn install, it'll prompt a msg and say use pnpm install.